lib/slack/web/faraday/response/raise_error.rb

```diff
@@ -6,7 +6,7 @@ module Response
         class RaiseError < ::Faraday::Response::Middleware
           def on_complete(env)
             raise Slack::Web::Api::Errors::TooManyRequestsError, env.response if env.status == 429
-            return unless (body = env.body) && !body['ok']
+            return if (body = env.body).nil? || body['ok'] || !env.success?
```

Comment:
Are we swallowing errors here when `!env.success?`? Or does that get raised as a Faraday error?

I would break this up for readability:

```ruby
return unless env.success?

body = env.body
return unless body
return if body['ok']
```